src/main/java/org/zone/region/flag/entity/player/move/message/display/title/TitleMessageDisplay.java

```diff
@@ -0,0 +1,39 @@
+package org.zone.region.flag.entity.player.move.message.display.title;
+
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.title.Title;
+import org.jetbrains.annotations.NotNull;
+import org.spongepowered.api.entity.living.player.Player;
+import org.zone.region.flag.entity.player.move.message.display.MessageDisplay;
+import org.zone.region.flag.entity.player.move.message.display.MessageDisplayType;
+import org.zone.region.flag.entity.player.move.message.display.MessageDisplayTypes;
+
+public class TitleMessageDisplay implements MessageDisplay {
+
+    private @NotNull Component subTitle;
+
+    public TitleMessageDisplay(@NotNull Component subTitle) {
+        this.subTitle = subTitle;
+    }
+
+    public TitleMessageDisplay() {}
```

Comment:
By calling this your making a @NotNull field ``null``

Reply:
the what how

Reply:
On like 19, you have a constructor that is blank, therefore all fields that have no value will then have a value of ``null``. This includes the fields that are labbled as ``@NotNull`` which then causes the issue of a not null field being null